driver-core/src/main/java/com/datastax/driver/core/Responses.java

```diff
@@ -448,6 +449,8 @@ private Rows(Metadata metadata, Queue<List<ByteBuffer>> data, ProtocolVersion ve
             public String toString() {
                 StringBuilder sb = new StringBuilder();
                 sb.append("ROWS ").append(metadata).append('\n');
+                // using default codec registry as this is only intended for debugging purposes
+                CodecRegistry codecRegistry = new CodecRegistry();
```

Comment:
Since CodecRegistry instances are inherently mutable, there isn't a default instance. However it could be interesting in cases like this to have such an immutable instance, to avoid unnecessary object allocation. I will git it a try.